x/concentrated-liquidity/position.go

```diff
@@ -649,11 +655,11 @@ func (k Keeper) RemovePositionIdToLock(ctx sdk.Context, positionId, underlyingLo
 // If lock is no longer active, returns false.
 func (k Keeper) PositionHasActiveUnderlyingLock(ctx sdk.Context, positionId uint64) (hasActiveUnderlyingLock bool, lockId uint64, err error) {
 	// Get the lock ID for the position.
+	// The only possible error retuned from `GetLockIdFromPositionId` is when position id to lock id mapping is not present.
+	// In this case, we silence the error and return false instead.
 	lockId, err = k.GetLockIdFromPositionId(ctx, positionId)
-	if errors.Is(err, types.PositionIdToLockNotFoundError{PositionId: positionId}) {
+	if err != nil {
 		return false, 0, nil
-	} else if err != nil {
-		return false, 0, err
 	}
```

Comment:
I don't think we should rely on that method never changing. The strictness here seems preferable.

If we proceed with the update and someone changes `GetLockIdFromPositionId` and doesn't check the callers in the future, a bug might get introduced

Reply:
I was wondering what was the reason we had this even though `GetLockIdFromPosition` only returns PositionIdToLockNotFoundError, thanks for claryfying